assets/javascript/app.js

```diff
@@ -1,185 +1,177 @@
 
-  $(document).ready(function(){
-
-    // Full height images toggle
-    $(".js-full-height").click(function( event ) {
-      $('body').toggleClass('full-height');
-    });
-
-    // Close all scenarios by default
-    $('.image-set-images').hide();
-    $('.toolbar').hide();
-
-
-
-    // toggle a scenario
-    $(".image-set-title").click(function( event ) {
-      $(this).toggleClass('open')
-      $(this).next().toggle();
-
-      // Show toolbar if at least one set is open
-      if ($('.image-set-title.open').length) {
-        $('.toolbar').show();
-      } else {
-        $('.toolbar').hide();
-      }
-
-      // If all are open, make sure toggle is set to 'Close all'
-      if ($('.image-set-title').length == $('.image-set-title.open').length){
-        console.log('all open!');
-        $(".js-close-all").show();
-        $(".js-open-all").hide();
-      }
-
-      // If all closed, make sure toggle is set to 'Open all'
-      if (!$('.image-set-title.open').length) {
-        console.log('all closed!');
-        $(".js-close-all").hide();
-        $(".js-open-all").show();
-      }
+$(document).ready(function(){
+  // Close all scenarios by default
+  $('.image-set-images').hide();
+  $('.toolbar').hide();
+
+  registerHandlers()
```

Comment:
All handlers moved to a separate method due to an issue with Handlebars. When we call the applyData again after receiving a response from the fetch call, the DOM elements are rendered, but the handlers aren't applied again, as they were originally being applied on document ready